### PR TITLE
added add_route_callback

### PR DESCRIPTION
**Pull request by ayberkenis**
changes made;

- added add_route_callback for easier usage in cogs

### usage example
```py
    async def callback(self, message: str):
        return {}

    async def cog_load(self) -> None:
        await self.ipc.start()
        await self.ipc.add_route_callback(self.callback, 'example_ipc_route')
```

- solved #6 

- ### also added errors module to the `__init__` to be able to use it in quart as it can be imported rn:

```py
@app.errorhandler(winerp.UnauthorizedError)
async def winerp_redirect(error):
    return redirect(url_for("app.index"))
```

### CODE REVIEW

**Review by AwesomeSam9523 — APPROVED** [review]
LGTM. Not tested as PR is made by trusted author.

**Comment by AwesomeSam9523** [comment]
@ayberkenis ready for merge?

**Comment by ayberkenis** [comment]
new commit has been pushed 4d20d0fe0cea19bb359cf1dee6a13dfa825ecf99

- [x] updated method name from `add_route_callback` to `add_route`

- [x] added docstring for methods that are not protected

- [x] events now has their own class

- [ ] add docs for object transfer

**Review by nouman0103 — APPROVED** [review]

File: winerp/__init__.py [hunk 1]
```diff
@@ -9,5 +9,6 @@
 """
 
 from .client import Client
-from .server import Server
+from .lib.errors import *
```
Comment (AwesomeSam9523) [inline winerp/__init__.py:12]: wildcard imports? lol

Comment (ayberkenis) [reply winerp/__init__.py:12]: oh.. forgot to change it...

**Review by AwesomeSam9523 — APPROVED** [review]
Good job on the Events part.

**Pull request merged**